neurons/miner.py

```diff
@@ -217,7 +214,7 @@ async def submit_model(self) -> None:
             return
```

Comment:
i guess here and in other validations we would need to also get rid of self.config.hd_repo_type